api/docs/docs/data.md

```diff
@@ -0,0 +1,210 @@
+
+### Infection Rate
+
+  R_t, or the estimated number of infections arising from a typical case.
+
+**Where to access**  
+* CSV column names: ``metrics.infectionRate``
+* JSON file fields: ``metrics.infectionRate``, ``metricsTimeseries.*.infectionRate``
+
+### Infection Rate Ci90
+
+  90th percentile confidence interval upper endpoint of the infection rate.
+
+**Where to access**  
+* CSV column names: ``metrics.infectionRateCI90``
+* JSON file fields: ``metrics.infectionRateCI90``, ``metricsTimeseries.*.infectionRateCI90``
+
+
+
+## Tests
+
+### Positive Tests
+
+  Cumulative positive test results to date
+
+**Where to access**  
+* CSV column names: ``actuals.positiveTests``
+* JSON file fields: ``actuals.positiveTests``, ``actualsTimeseries.*.positiveTests``
+
+### Negative Tests
+
+  Cumulative negative test results to date
+
+**Where to access**  
+* CSV column names: ``actuals.negativeTests``
+* JSON file fields: ``actuals.negativeTests``, ``actualsTimeseries.*.negativeTests``
+
+### Test Positivity Ratio
+
+  Ratio of people who test positive calculated using a 7-day rolling average.
+
+**Where to access**  
+* CSV column names: ``metrics.testPositivityRatio``
+* JSON file fields: ``metrics.testPositivityRatio``, ``metricsTimeseries.*.testPositivityRatio``
+
+### Test Positivity Ratio Details
+
+  
+
+**Where to access**  
+* CSV column names: ``metrics.testPositivityRatioDetails``
+* JSON file fields: ``metrics.testPositivityRatioDetails``, ``metricsTimeseries.*.testPositivityRatioDetails``
+
+
+
+## Hospitalizations
+
+### Icu Beds
+
+  Information about ICU bed utilization details.
+
+Fields:
+ * capacity - Current staffed ICU bed capacity.
+ * currentUsageTotal - Total number of ICU beds currently in use
+ * currentUsageCovid - Number of ICU beds currently in use by COVID patients.
+ * typicalUsageRate - Typical ICU utilization rate.
+
+**Where to access**  
+* CSV column names: ``actuals.icuBeds``
+* JSON file fields: ``actuals.icuBeds``, ``actualsTimeseries.*.icuBeds``
+
+### Hospital Beds
+
+  Information about acute bed utilization details.
+
+Fields:
+ * capacity - Current staffed acute bed capacity.
+ * currentUsageTotal - Total number of acute beds currently in use
+ * currentUsageCovid - Number of acute beds currently in use by COVID patients.
+ * typicalUsageRate - Typical acute bed utilization rate.
+
+**Where to access**  
+* CSV column names: ``actuals.hospitalBeds``
+* JSON file fields: ``actuals.hospitalBeds``, ``actualsTimeseries.*.hospitalBeds``
+
+### Icu Capacity Ratio
+
+  Ratio of staffed intensive care unit (ICU) beds that are currently in use.
+
+**Where to access**  
+* CSV column names: ``metrics.icuCapacityRatio``
+* JSON file fields: ``metrics.icuCapacityRatio``, ``metricsTimeseries.*.icuCapacityRatio``
+
+### Icu Headroom Ratio
+
+  
+
+**Where to access**  
+* CSV column names: ``metrics.icuHeadroomRatio``
+* JSON file fields: ``metrics.icuHeadroomRatio``, ``metricsTimeseries.*.icuHeadroomRatio``
```

Comment:
Wondering whether it makes sense to keep all ICU stuff together and move this up. I understand how it's currently organized (bed counts first and then the the ratios), but maybe worth considering the change.